plugins/tiddlywiki/aws/templates/lambda/sjcl.tid

```diff
@@ -2,8 +2,6 @@ title: $:/plugins/tiddlywiki/aws/lambda/sjcl
 
 \rules only filteredtranscludeinline transcludeinline
```

Comment:
The anonymous wrapper should not be removed for this template. We need it to hide the global overrides.

Reply:
That was the reason, why I did create 1 PR per plugin as a start. It will be much easier to review.

Reply:
Thanks @pmario if you can revert this tiddler I am happy to merge this.